docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -51,19 +51,8 @@ When we added the `kubernetes` extension to the command line invocation above, t
     </dependency>
 ----
 
-By adding this dependency, we now have the ability to configure the Kubernetes resource generation and application using the usual `application.properties` approach that Quarkus provides.
-The configuration items that are available can be found in: `io.quarkus.kubernetes.deployment.KubernetesConfig` class.
-Furthermore, the items provided by `io.quarkus.deployment.ApplicationConfig` affect the Kubernetes resources.
-
-By using the following configuration for example:
-
-[source]
-----
-quarkus.kubernetes.group=yourDockerUsername # this is optional and defaults to your username if not set.
-quarkus.application.name=test-quarkus-app # this is also optional and defaults to the project name if not set
-----
-
-and following the execution of `./mvnw package` you will notice amongst the other files that are created, two files named
+By adding this dependency, we enable the generation of kubernetes manifests each time we perform a build.
```

Comment:
I think we should probably use `Kubernetes` (upper-case `K`) in all instances of the doc that aren't referring to code